core/core-impl/src/main/java/ai/timefold/solver/core/impl/heuristic/selector/move/generic/list/kopt/MultipleDelegateList.java

```diff
@@ -42,6 +43,7 @@ public MultipleDelegateList(List<T>... delegates) {
         }
 
         this.totalSize = sizeSoFar;
+        this.cachedRebaseResult = null;
```

Comment:
Maybe add a comment here as to why it's safe to store this here?

Reply:
I think it not safe, due to the fact the accepted move is rebased twice (one for it score director, and one for the solver score director); I can move the cached result to KOptListMove

Reply:
Go for it. I think it makes more sense to store it there, because that is where `rebase` is.